x-pack/plugins/security/public/components/confirm_modal.tsx

```diff
@@ -48,48 +47,40 @@ export const ConfirmModal: FunctionComponent<ConfirmModalProps> = ({
   ownFocus = true,
```

Comment:
__nit:__ it looks like `ownFocus` is no longer needed, would you mind removing it from here, from `ConfirmModalProps` and from the component JSDoc?